modules/smartadserverBidAdapter.js

```diff
@@ -12,6 +12,7 @@ import {
 import {
   createEidsArray
 } from './userId/eids.js';
+import { defaultConfig } from 'sinon';
```

Comment:
What's the use of this import ?

Reply:
This import is useless, i do not know why it is here and it should have been removed.